docs/content/docs/internals/job_scheduling.md

```diff
@@ -95,3 +95,22 @@ For that reason, the execution of an ExecutionVertex is tracked in an {{< gh_lin
 {{< img src="/fig/state_machine.svg" alt="States and Transitions of Task Executions" width="50%" >}}
 
 {{< top >}}
+
+## Repeatable Resource Cleanup Strategy
```

Comment:
I guess, we concluded (when talking with @knaufk ) that we would move this part into `Deployment / Overview` before `Vendor Solutions`. But looking into the docs again: I feel like we could put it right before the `Deployment Modes` section and connect it semantically with the `External Components` table entries (because cleanup means cleaning up all external components). WDYT?

Reply:
Ah, that's a great idea. I will move it into there and connect it with the external components section.